SimG4CMS/Calo/interface/CaloHitID.h

```diff
@@ -25,6 +25,11 @@ class CaloHitID {
   void setID(uint32_t unitID, double timeSlice, int trackID, uint16_t depth = 0);
   void reset();
 
+  void overwriteTrackID(int trackID) { theTrackID = trackID; }
+  bool hasFineTrackID() const { return theFineTrackID != -1; }
+  void setFineTrackID(int fineTrackID) { theFineTrackID = fineTrackID; }
+  int getFineTrackID() const { return hasFineTrackID() ? theFineTrackID : theTrackID; }
```

Comment:
getter name should not include `get` (http://cms-sw.github.io/cms_coding_rules.html 2.11)

Reply:
Fixed througout